mtags/src/main/scala/scala/meta/internal/mtags/JavaToplevelMtags.scala

```diff
@@ -27,6 +27,14 @@ class JavaToplevelMtags(val input: Input.VirtualFile) extends MtagsIndexer {
     }
   }
 
+  def readPackage: List[String] = {
+    fetchToken
```

Comment:
```suggestion
    fetchToken // start of file
```
maybe useful to comment?